src/js/pages/exercises.js

```diff
@@ -3,13 +3,28 @@ import { renderExercises } from '../renderExercises';
 import exercisesTemplate from '../../exercises.html?raw';
 
 const cardsContainer = document.querySelector('.cards-container');
+export const exercisesElement = document.createElement('div');
+exercisesElement.innerHTML = exercisesTemplate;
 
-export const getExercises = async (params) => {
-  const exercises = await api.exercises.getExercises(params);
-  const markup = await renderExercises(exercises);
+const categoryMap = {
+  'body parts': 'bodypart',
+  muscles: 'muscles',
+  equipment: 'equipment',
+};
 
-  cardsContainer.innerHTML = `<ul class="main-exercises">${markup}</ul>`;
+export const getExercises = async (params) => {
+  const exercises = await api.exercises.getExercises(normalizeParams(params));
```

Comment:
it's more update parasms

Reply:
This is an api method call. It needs all the parameters